assets/javascripts/discourse/templates/components/chat-live-pane.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if (and fullPage includeHeader)}}
-  <div class="chat-full-page-header {{if previewing "previewing"}}">
+  <div class="chat-full-page-header {{if previewing 'previewing'}}">
```

Comment:
For now please fix it manually @martin-brennan, it's on @cvx plate to get linting on par between core and chat